### DISCUSSION TITLE
`BezierArrow`s (or possibly adding arrow tips to `CubicBezier`)

### DISCUSSION THREAD

**Discussion by egidioln** [ManimCommunity/manim#3028] [Ideas]
Hi everyone.

 Recently I had to use BezierArrows to draw a graph like this
![image](https://user-images.githubusercontent.com/12188889/202772648-607907f8-5605-4fa9-a3a0-d298415a5f8b.png)

I  worked that around by adding a small arrow at the endpoint of the `CubicBezier`. However, this was not optimal and I was wondering if this shouldn't be a feature of `CubicBezier` or even if there shouldn't exist something like `BezierArrow`. I can try to code it if you think it's needed :)

All the best!

**Comment by MrDiver** [comment]
I think this will be easier to do if we changed the structure of tipable, this is something im looking forward to do but i currently do not have the ressources to change that. But i think this will be a result of the change automagically or make it a lot easier to implement this kind of behavior.
